src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.motors.PIDSparkMax;
+import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+@RobotMapFor("Gaston")
+public class GastonMap extends RobotMap {
+    private final DigitalInput outsideLimit = new DigitalInput(4);
+    private final DigitalInput insideLimit = new DigitalInput(5);
+
+    // initializes relative encoder and pid controller, we don't need the encoder rn
+
+    private PIDSparkMax deploymentMotor = new PIDSparkMax(2, MotorType.kBrushless);
+
+    // private RelativeEncoder deploymentEncoder =
+    // deploymentMotor.getEncoder().getRaw();
+    private final SparkMaxPIDController deploymentPidController = deploymentMotor.getPidController();
```

Comment:
All of these should be defined inside of `getIntakeMap`, no need to have them at the class level.